### PR TITLE
install/windows/index.xml: add the `option` tag

### PR DESCRIPTION
**Pull request by mmalferov**

### CODE REVIEW

File: install/windows/index.xml [hunk 1]
```diff
@@ -57,7 +57,8 @@
   </para>
 
   <para>
-   The CRT installer supports the /quiet and /norestart command-line switches, so you can script running it.
+   The CRT installer supports the <command>/quiet</command> and <command>/norestart</command>
```
Comment (cmb69) [inline install/windows/index.xml:60]: I think that should better be:
```suggestion
   The CRT installer supports the <option>/quiet</option> and <option>/norestart</option>
```

However, I'm not even sure whether these are still supported by the CRT installers.

Comment (mmalferov) [reply install/windows/index.xml:60]: These are still supported ;-)

https://learn.microsoft.com/en-us/cpp/windows/redistributing-visual-cpp-files?view=msvc-170#command-line-options-for-the-redistributable-packages

I changed the tags from `command` to 'options'

Comment (cmb69) [reply install/windows/index.xml]: Thank you for checking this!

**Comment by cmb69** [comment]
Thank you!

**Pull request merged**